src/index.js

```diff
@@ -61,7 +62,7 @@ export default function (CustomElement, opts) {
       });
     }
     render() {
-      return React.createElement(tagName, null, this.props.children);
+      return React.createElement(tagName, { style: this.props.style }, this.props.children);
```

Comment:
Yeah, you can just do `<Element {...this.props} />` and everything will get set through our `willReceiveProps()` callback (which is what we use to ensure props are set as properties on the web component).